src/arch/intsimdmatrixavx2.cpp

```diff
@@ -80,24 +80,80 @@ static inline void MultiplyGroup(const __m256i& rep_input, const __m256i& ones,
   result = _mm256_add_epi32(result, weights);
 }
 
-// Extracts and converts 8x32-bit results from result, adding the bias from wi
-// and scaling by scales, before storing in *v. Note that wi, scales and v are
-// expected to contain 8 consecutive elements or num_out if less.
-static inline void ExtractResults(__m256i& result, __m256i& shift_id,
-                                  const int8_t*& wi, const double*& scales,
-                                  int num_out, double*& v) {
-  for (int out = 0; out < num_out; ++out) {
-#ifndef _MSC_VER
-    auto res = _mm256_extract_epi32(result, 0);
+// Load 64 bits into the bottom of a 128bit register.
+// We don't actually care what the top 64bits are, but this ends
+// up with them being zero.
+static inline __m128i load64_to_128(const int8_t *wi_)
+{
+#if 1
```

Comment:
Can `#if 1` and the `#else` part be removed?

Reply:
Fixed.